Libraries/LibWeb/MathML/MathMLElement.cpp

```diff
@@ -59,4 +65,60 @@ void MathMLElement::visit_edges(JS::Cell::Visitor& visitor)
     HTMLOrSVGElement::visit_edges(visitor);
 }
 
+bool MathMLElement::is_presentational_hint(FlyString const& name) const
+{
+    return first_is_one_of(name, "dir", "mathcolor", "mathbackground", "mathsize", "displaystyle", "scriptlevel");
```

Comment:
Regarding `MathML::AtributeNames`, the benefit is that those would be `FlyString`s, meaning that comparing them is much faster.